tests/test_spec.py

```diff
@@ -77,7 +83,7 @@ def lone_get():
         pass
 
     @app.route('/lone', methods=['POST'])
-    @api.validate(json=ExampleModel, resp=Response(HTTP_200=ExampleModel, HTTP_404=None))
+    @api.validate(json=ExampleModel, resp=Response(HTTP_200=ExampleNestedList, HTTP_400=ExampleNestedModel))
```

Comment:
is it possible to change the interface to Response to allow for us to put `List[ExampleNestedModel]` instead of sub_classing?

Reply:
Do we definitely want a list as the root element?

Reply:
Personally I don't think we do, maybe we can put something on the Model that says the schema that it produces should be as a list?

Reply:
Hmm.  Might be slightly hacky to do.  I think I'm happy to represent a list as a model type.  Then there's a one to one mapping between model and schema.

Reply:
I think we need to support a list as root element in spectree, because it's valid openapi.  Whether we want to do it in our own code is another matter.